unittests/backends/braket/BraketTester.cpp

```diff
@@ -0,0 +1,318 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+#include "CUDAQTestUtils.h"
+#include "common/FmtCore.h"
+#include "cudaq/algorithm.h"
+#include <fstream>
+#include <gtest/gtest.h>
+#include <regex>
+
+// port number and localhost connect to mock_qpu backend server within the
+// container (mock_qpu/braket).
+std::string mockPort = "5000";
```

Comment:
mock port 5000 is already used by the anyon tester. Please use 62444, and set the anyone tester to 62445.

Reply:
Fixed in aa33688398acc6b400fdc8e5e92b037d3fe794c0. Will change anyon port in separate PR.

Reply:
This should probably just be typed as `constexpr const char mockPort[] = ...` to avoid constructor overhead at load time.